### ISSUE TITLE
next/previous link for non-english websites and translation result page suggestion

### ISSUE THREAD

**Issue by klesh** [brookhong/Surfingkeys#175]
Surfingkeys is awesome!

I would like to suggest some improvement:

1. The next/previous regex should be able to be overrided in customize settings which will be very convenience for Non-English websites browsing.

2. It would be nice if there is a loading indicator for translation. For the result page, pronunciations and audios would be helpful. I had my customization you may like to take a look:

```js
mapkey('Q', '#8Open omnibar for word translation', function() {
    Front.openOmniquery({
        url: "https://api.shanbay.com/bdc/search/?word=",
        query: Visual.getWordUnderCursor(),
        style: "opacity: 0.8;",
        parseResult: function(res) {
            var res = eval("a=" + res.text);
            if (res.data.definition) {
                var tmp = [];
                for (var reg in res.data.pronunciations) {
                    tmp.push(`[${reg}] ${res.data.pronunciations[reg]}`);
                    tmp.push(`<audio src="${res.data[reg+'_audio']}" controls></audio>`);
                }
                tmp.push(res.data.definition);
                return [ `<pre>${tmp.join('\n')}</pre>` ];
            }
            return [res.data.definition || res.msg];
        }
    });
});
```

PS: I'd like to contribute code directly if you accept contributor.

**Comment by brookhong** [comment]
https://github.com/brookhong/Surfingkeys/commit/3f019c9d807ba208d2ec64f9a38386b7f4bc5450

Thanks for suggestion.
Your code is merged. Feel free to post PR next time, the PR that makes sense will be merged as you can see from https://github.com/brookhong/Surfingkeys/pulls?q=is%3Apr+is%3Aclosed

**Comment by brookhong** [comment]
Released in 0.6.9

For regex to match next/previous links, please use settings below:

    settings.prevLinkRegex = /((上页|上一页|<<|prev(ious)?)+)/i;
    settings.nextLinkRegex = /((下页|下一页|>>|next)+)/i;
